common/djangoapps/tedix_ro/views.py

```diff
@@ -274,7 +274,14 @@ def user_due_date_data(user, due_date):
 def my_reports_main(request):
     if not request.user.is_authenticated():
         return redirect(get_next_url_for_login_page(request))
-    return render_to_response('my_reports_main.html')
+
+    # Account activation message
+    account_activation_messages = [
+        message for message in messages.get_messages(request) if 'account-activation' in message.tags
+    ]
+    context = {'account_activation_messages': account_activation_messages}
```

Comment:
messages are already in the context of each django template, you just need to render them

https://docs.djangoproject.com/en/2.2/ref/contrib/messages/